innkeeper/client.go

```diff
@@ -374,14 +409,31 @@ func (c *Client) requestData(authRetry bool, url string) ([]*routeData, error) {
 	return result, err
 }
 
+func (c *Client) LoadAndParseAll() ([]*eskip.RouteInfo, error) {
+	routeList, err := c.LoadAll()
+	if err != nil {
+		return nil, err
+	}
+
+	routeInfos := []*eskip.RouteInfo{}
+
+	for _, route := range routeList {
+		routeInfo := &eskip.RouteInfo{
+			*route, nil}
```

Comment:
I guess it's up to the one who works at #83 to figure it out. In case everything is always perfect, we will remove the RouteInfo as you suggested. The goal of this #83 ticket and other tickets I created is to extract the non critical stuff out of this already monolithic pull request. So we can deal with each smaller problem individually after we merge this one.